pom.xml

```diff
@@ -38,6 +38,11 @@
             <version>${graylog2.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>23.6-jre</version>
+        </dependency>
```

Comment:
Please migrate the POM to use the official `graylog-plugin-parent` parent POM.
This way you don't have to manually add any dependencies which are pulled in by Graylog itself.

Example: https://github.com/graylog-labs/graylog-plugin-internal-logs/blob/2.4.0-beta.2/pom.xml